source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+---
+title: Data Protection for GOV.UK Account users
+weight: 32
```

Comment:
Change the weight to an unused value so the content appears in the correct order

Reply:
Done